lib/components/artist/shows/metadata.js

```diff
@@ -33,6 +34,10 @@ class Metadata extends React.Component {
   dateAndLocationString(show) {
     return show.location.city? <SerifText style={styles.serifText, {color: 'grey'}}>{show.location.city + ', ' + show.exhibition_period}</SerifText> : null;
   }
+
+  statusText(show) {
+    return show.status_update? <SerifText style={{color: show.status_update.match(/opening/i) ? colors['green-regular'] : colors['red-regular']}}>{show.status_update}</SerifText> : null;
```

Comment:
Instead of matching against a human-readable message, you could use the `status` field here to compare against.